sycl/test-e2e/Adapters/max-local-mem-size.cpp

```diff
@@ -21,14 +21,14 @@ int main() {
     auto LocalSize =
         Q.get_device().get_info<sycl::info::device::local_mem_size>();
     Q.submit([&](sycl::handler &cgh) {
-       auto LocalAcc = sycl::local_accessor<float>(LocalSize + 1, cgh);
+       auto LocalAcc = sycl::local_accessor<char>(LocalSize + 1, cgh);
        cgh.parallel_for(sycl::nd_range<1>{32, 32}, [=](sycl::nd_item<1> idx) {
          LocalAcc[idx.get_global_linear_id()] *= 2;
        });
      }).wait();
   } catch (const std::exception &e) {
     std::puts(e.what());
   }
-  // CHECK-ZERO: Local memory for kernel exceeds the amount requested using SYCL_PI_CUDA_MAX_LOCAL_MEM_SIZE
+  // CHECK-ZERO: Invalid value specified for
```

Comment:
this seems incomplete

Reply:
It's incomplete on purpose because after that it prints the environment variable which is different for CUDA/HIP, so it's slightly annoying to match, and this should be enough to ensure an error message is emitted

Reply:
ah right, makes sense